Formula/ncspot.rb

```diff
@@ -32,7 +32,7 @@ class Ncspot < Formula
   def install
     ENV["COREAUDIO_SDK_PATH"] = MacOS.sdk_path_if_needed
     system "cargo", "install", "--no-default-features",
-                               "--features", "portaudio_backend,cursive/pancurses-backend,share_clipboard",
+                               "--features", "portaudio_backend,termion_backend,share_clipboard",
```

Comment:
Since you say this is the default, will it work if we leave it out?

Reply:
Just tested the build without specifying a backend (i.e. `"--no-default-features", "--features", "portaudio_backend,share_clipboard"`).

While the build process didn't throw an error, the built binary didn't work properly: no TUI was created.

Reply:
Yes, with `--no-default-features` no terminal backend will be included unless explicitly specified :)